src/options.jl

```diff
@@ -18,4 +18,12 @@ Base.@kwdef struct Options
     align_pair_arrow::Bool = false
     conditional_to_if::Bool = false
     normalize_line_endings::String = "auto"
+    for_in_replacement::String = "in"
+end
+
+function valid_for_in_op(s::String)
```

Comment:
```suggestion
valid_for_in_op(s::String) = s ∈ ("in", "=", "∈")
```

And... maybe we can run this function in constructor of `Options`?

Reply:
that makes a lot of sense